tests/commands/test-sdk.js

```diff
@@ -0,0 +1,802 @@
+					addAnalyticsEvent: (name, obj) => {}
+				};
+
+				SDK.run(logger, config, cli, function (err) {
+					if (err) {
+						return finished(err);
+					}
+
+					logger.calls[0].should.eql([ 'banner', undefined ]);
+					logger.calls[1].should.eql([ 'log', `Titanium SDK ${'9.2.0.GA'.cyan} is already installed!\n` ]);
+					logger.calls[2].should.eql([ 'log', `Run '${('titanium sdk install 9.2.0.GA --force').cyan}' to re-install.\n` ]);
+
+					finished();
+				});
+			});
+
+			it('with uninstalled sdk', function (finished) {
+				this.timeout(5 * 60 * 1000); // 5 minutes
+				const cli = {
+					argv: {
+						_: [ 'install' ],
+						$: 'titanium',
+						default: false,
+						force: false,
+						quiet: false,
+						'progress-bars': true,
+						version: '7.5.0.GA'
+					},
+					env: {
+						installPath: sdksPath,
+						os: {
+							name: 'osx',
+							sdkPaths: [ sdksPath ],
+						},
+						sdks: {
+							'9.2.0.GA': {
+								manifest: {
+									name: '9.2.0.v20200923092031',
+									version: '9.2.0.GA',
+								},
+								path: path.join(sdksPath, 'mobilesdk/osx/9.2.0.GA')
+							}
+						}
+					},
+					addAnalyticsEvent: (name, obj) => {}
+				};
+
+				SDK.run(logger, config, cli, function (err) {
+					if (err) {
+						return finished(err);
+					}
+
+					try {
+						logger.calls[0].should.eql([ 'banner', undefined ]);
+						logger.calls[1].should.eql([ 'log', `Downloading ${'https://builds.appcelerator.com/mobile-releases/7.5.0/mobilesdk-7.5.0.GA-osx.zip'.cyan}` ]);
+						logger.calls[2].should.eql([ 'log', '\n' ]); // end of progress bar
+						logger.calls[3].should.eql([ 'log', `Extracting SDK to ${sdksPath.cyan}` ]);
+						logger.calls[4].should.eql([ 'log', '\n' ]); // end of progress bar
+						logger.calls[5].should.eql([ 'log', `Titanium SDK ${'7.5.0.GA'.cyan} successfully installed!\n` ]);
+					} catch (error) {
+						return finished(error);
+					} finally {
+						// Remove the extracted contents
+						try {
+							fs.removeSync(path.join(sdksPath, 'modules'));
+							fs.removeSync(path.join(sdksPath, 'mobilesdk/osx/7.5.0.GA'));
+						} catch (error) {
+							// squash
+						}
+					}
+					finished();
+				});
+			});
+
+			it('with pre-existing local sdk, matching timestamp, without --force', function (finished) {
+				const cli = {
+					argv: {
+						_: [ 'install' ],
+						$: 'titanium',
+						default: false,
+						force: false,
+						quiet: false,
+						'progress-bars': false,
+						version: path.join(__dirname, '../resources/9.2.0.GA.zip')
+					},
+					env: {
+						installPath: sdksPath,
+						os: {
+							name: 'osx',
+							sdkPaths: [ sdksPath ],
+						},
+						sdks: {
+							'9.2.0.GA': {
+								manifest: {
+									name: '9.2.0.v20200923092031',
+									version: '9.2.0.GA',
+									timestamp: '9/23/2020 16:25'
+								},
+								path: path.join(sdksPath, 'mobilesdk/osx/9.2.0.GA')
+							}
+						}
+					},
+					addAnalyticsEvent: (name, obj) => {}
```

Comment:
<!--
  0 failure: 
  2 warning:  tests/commands/te..., tests/commands/te...
  
  
  DangerID: danger-id-Danger;
  File: tests/commands/test-sdk.js;
  Line: 448;
-->

- :warning: tests/commands/test-sdk.js line 448 – 'name' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)
- :warning: tests/commands/test-sdk.js line 448 – 'obj' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)